superset/templates/appbuilder/navbar_right.html

```diff
@@ -41,9 +44,18 @@
   <a
     tabindex="-1"
     href="{{ documentation_url }}"
-    title="Documentation"
+    title="{{ documentation_text }}"
+    target="_blank"
   >
+  {% if documentation_icon %}
+    <img
+      width="{{ documentation_icon_width }}"
```

Comment:
seems like this wouldn't be necessary. Maybe we could set `width` to 100% or something? Or just enforce that the icon people use must be a certain size (64px or 128px or something like that)

Reply:
Set to 100% 👍